src/components/Filter/FilterInterfaces.tsx

```diff
@@ -7,7 +7,7 @@ export interface FilterInterface {
 export interface FilterListInterface {
   field?: string | string[] | undefined;
   operation?: string | string[] | undefined;
-  value?: string | string[] | undefined;
+  value?: any;
```

Comment:
I think any will cause the linter to complain (warning not error) but aside from that i think this is fine - but are we expecting a `string | string[] | undefined` to come back?

Reply:
Probably no, this value can be boolean, string, different types of object, number, basically it can be anything as this value would be the value that would filter the content.

Reply:
ahh i see but since the user is inputing their value using a textbox - wouldnt it always start out as a string? or are is the value being cast to something else by the time it gets into `value`?

Reply:
Not every time, for exemple when user selects if the value is between or not between we are calling the `RangeInput` from Number.js and in this case the value returns as an object `{max: 10, min: 1}` the same with is empty, that the value is returning as a boolean `true | false`.

Reply:
ahh ok